### ISSUE TITLE
Netflow ingestion - "Error template not found"

### ISSUE THREAD

**Issue by Sonicscream** [akvorado/akvorado#1112]
### Discussed in https://github.com/akvorado/akvorado/discussions/1057

<div type='discussions-op-text'>

<sup>Originally posted by **Sprooty** January 24, 2024</sup>
Hi all,

I have hit a roadblock trying to get Akvorado to ingest opensense data, i can see the data is mkaing it to the container `"[docker-akvorado-inlet-1]"`

All i am seeing in the logs is,
`ERR netflow/root.go:198 > error while decoding NetFlow/IPFIX | error=NetFlowV9 NetFlowV9 [version:9 type:Decode obsDomainId:0: templateId:259] Error template not found`

I have trawled the doco and can't find any reference to if/how i am to specify a template.

any help would be greatly appreciated 
</div>

**Comment by vincentbernat** [comment]
Issues are for bugs.

**Comment by folkertvanheusden** [comment]
If that is not a bug, how can I resolve it? I'm seeing it as well with opensense.

**Comment by Sonicscream** [comment]
In addition to OPNsense, I've attempted to use other netflow exporters, such as softflowd. Nothing works. I've tried all combinations of settings. I've even gotten other tools than akvorado to read the netflow exports.

It might be that we didn't follow the documentation close enough, but I've literally wasted hours trying to find what I didn't do correctly according to the documentation.

Until a dev can shine a light on this issue, I won't be able to use akvorado.
